portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java

```diff
@@ -34,7 +34,7 @@
 
 /** Servlet that deletes bucket list content*/
 @WebServlet("/delete-data")
-public class DeleteDataServlet extends HttpServlet 
+public class DeleteDataServlet extends HttpServlet {
```

Comment:
So I had just merged multiple other PRs into master, and then pulled those before starting development on this branch, so I thought I'd make the few design changes that emerged from merging those PRs, on this branch. I thought it wouldn't be worth it to make a whole new branch for a couple lines of code, but I probably just should have because yeah, this has nothing to do with the rest of the PR.